src/pages/Product/ProductList.jsx

```diff
@@ -1,32 +1,40 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { Navbar } from '../../components'
 import './Product.css'
-import { products } from '../../backend/db/products'
 import { Filter } from './component/Filter'
-import {Link} from 'react-router-dom'
+import axios from 'axios'
 
 export const ProductList = () => {
+  const [res, setRes] = useState([])
+
+  const dataFetch = async () => {
+    const response = await axios.get('/api/products')
+    setRes(response.data.products)
+  }
+
+  useEffect(dataFetch, [])
+
   return (
-    
     <>
-    <main className="product_page">
-      <Navbar LoginOrSignup="Login" address='/login-page' />
-        <section class="all_product content">
-        <Filter />
+      <main className="product_page">
+        <Navbar LoginOrSignup="Login" address="/login-page" />
+        <section className="all_product content">
+          <Filter />
           <aside className="product_list">
-          <h3 className='productHeading'>Showing All Products</h3>
+            <h3 className="productHeading">Showing All Products</h3>
             <section className="product_list_section">
-              {products.map((productItem) => {
+              {res.map((productItem) => {
                 return (
                   <div className="product_list_item">
                     <img
                       className="productListImg"
-                      src={productItem.productImg} alt='images'
+                      src={productItem.productImg}
```

Comment:
you could destructure these items so that you don't need to write products.itemName

Example: res.map(({productImage}))

Reply:
I will change it.